lib/ansible/plugins/lookup/url.py

```diff
@@ -69,24 +69,24 @@
     version_added: "2.10"
     default: ansible-httpget
     vars:
-        - name: ansible_lookup_url_agent
+        - name: ansible_lookup_url_http_agent
     env:
-        - name: ANSIBLE_LOOKUP_URL_AGENT
+        - name: ANSIBLE_LOOKUP_URL_HTTP_AGENT
```

Comment:
these are not typos but changes to the variables and entries used

Reply:
if you wan to make the case that new ones are needed/better, that is one thing, but cannot remove existing w/o a deprecation period

Reply:
I just thought it might be a typo, since the key doesn't match the option name, and I didn't find where the key `agent` is defined. I just want to fix the typo with the pull request. Add a new variable name is out of scope.

Reply:
the documentation you are amending ALSO defines the actual variables and keys used, they are one and the same

Reply:
Oh, I see. The doc itself serves as the definition. Thanks for the explanation.

Reply:
Anyway, I've reverted this incorrect change. Please take another look when you have time. Thanks.

Reply:
Also, not typos, the options/variables are not required to EXACTLY match the options named for them, for example for connection plugins `remote_addr` normally is mapped to `ansible_host` and `inventory_hostname` variables.